planning/README.md

```diff
@@ -26,6 +26,20 @@ You can verify the scores of your submissions on the [Challenges website][challe
 
 [challenges]: https://challenges.duckietown.org/v4/humans/challenges
 
+## ℹ️️ Check Robot Compatibility
+
+While we try our best to support running these exercises on all versions of the Duckiebot, some activities require special hardware and
+are only supported on specific robots. Please use this section to ensure the compatibility of the exercise and your
+robot.
+
+The support matrix of this exercise is as follows:
+
+| Duckiebot Type   	                                                                                | Configuration 	 | Support Level   	    |
+|---------------------------------------------------------------------------------------------------|-----------------|----------------------|
+| [DB21-J4](https://get.duckietown.com/products/duckiebot-db21?variant=41543707099311)            	 | Jetson 4GB    	 | ✔️ Full Support    	 |
+| [DB21-J2](https://get.duckietown.com/products/duckiebot-db21?variant=40700056830127)            	 | Jetson 2GB    	 | ✔️ Full Support 	    |
+
+
```

Comment:
The exercise `planning` does not run on the physical robot, you can remove this.